resources/views/patients/create.blade.php

```diff
@@ -3,56 +3,99 @@
+                                placeholder="********">
+                        </div>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="phone" class="form-label">Phone Number *</label>
+                            <input type="text" class="form-control" id="phone" name="phone"
+                                placeholder="09*********">
+                        </div>
                     </div>
-                    <div class="mb-3">
-                        <label for="gender" class="form-label">Gender</label>
-                        <div class="form-group d-flex">
-                            <div class="form-check">
-                                <input class="form-check-input" type="radio" name="gender" id="male">
-                                <label class="form-check-label me-3" for="male">
-                                    Male
-                                </label>
-                            </div>
-                            <div class="form-check">
-                                <input class="form-check-input" type="radio" name="gender" id="female" checked>
-                                <label class="form-check-label me-3" for="female">
-                                    Female
-                                </label>
-                            </div>
-                            <div class="form-check">
-                                <input class="form-check-input" type="radio" name="gender" id="other">
-                                <label class="form-check-label" for="other">
-                                    Other
-                                </label>
+                    <div class="col-lg-6 col-md-6">
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="age" class="form-label">Age *</label>
+                            <input type="number" class="form-control" id="age" name="age" placeholder="age">
+                        </div>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="address" class="form-label">Address *</label>
+                            <input type="text" class="form-control" id="address" name="address" placeholder="address">
+                        </div>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="blood_type" class="form-label">Blood Type</label>
+                            <select class="form-select" aria-label="Default select example" id="blood_type"
+                                name="blood_type">
+                                <option selected>Select Blood Type</option>
+                                <option value="1">A</option>
+                                <option value="2">O</option>
+                                <option value="3">B</option>
+                                <option value="4">AB</option>
```

Comment:
I don't know your blood_type ui order?
in your migration file : $table->enum('blood_type', ['A','B', 'AB','O'])->nullable()->comment('blood_type');
So, your order is 'A','B', 'AB','O'.